keyboards/capsunlocked/cu80/v2_ansi/readme.md

```diff
@@ -0,0 +1,15 @@
+# CU80 v2 ANSI
+
+The CU80 v2 ANSI is the PCB for round 2 of the CU80 TKL keyboard, with fixed layout ANSI and VIA support.
+
+To place the keyboard into bootloader mode in order to flash it, hold the ESC key while plugging the board in.
```

Comment:
Is there a reset button on the PCB? In case a user manages to mess up Bootmagic.

Reply:
There is indeed a fixed reset button on the PCB